src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,9 @@
+$watch-size: 700px;
+$center-size: 50px;
+$hands-width: 15px;
+$half-hend-width: $hands-width / 2;
+$seconds-hand-langht: $watch-size / 2 - 50px;
+$seconds-hand-top: -$seconds-hand-langht + $center-size / 2;
```

Comment:
There's a typo in the variable name, it should be `$seconds-hand-length` instead of `$seconds-hand-langht`. Please update the variable usage.